src/histolab/filters/image_filters.py

```diff
@@ -620,6 +630,8 @@ class RagThreshold(ImageFilter):
     the segments, combine similar regions based on threshold value,
     and then output these resulting region segments.
 
+    .. figure:: https://user-images.githubusercontent.com/31658006/116543569-506dfb80-a8ee-11eb-9d59-413b33de8e73.jpeg
```

Comment:
this image needs to be done again, as both the text and the image are misaligned between the two steps